src/js/zarith.js

```diff
@@ -452,7 +462,17 @@ function ml_z_to_bits(z1, z2) {
 }
 
 // external of_bits: string -> t = "ml_z_of_bits"
-// Should provide ml_z_of_bits const
-function ml_z_of_bits(z1, z2) {
-
+//Provides: ml_z_of_bits
```

Comment:
Should this be marked `const`?

Reply:
Yes. I haven't added those annotations yet but I'll do it.